src/ngraph_encapsulate_op.cc

```diff
@@ -541,6 +579,8 @@ class NGraphEncapsulateOp : public OpKernel {
   std::vector<bool> m_input_is_static;
   std::mutex m_compute_lock;
   string m_op_backend_name;
+  std::list<std::string> LRU;
+  int NGRAPH_TF_FUNCTION_CACHE_ITEM_DEPTH = 16;
```

Comment:
We could move `const char* cache_depth_specified = std::getenv("NGRAPH_TF_FUNCTION_CACHE_ITEM_DEPTH");` here
(but then we wont be able to tweak cache size while the model is running).

Pros and Cons for doing it in `Compute` vs in the constructor?

Reply:
Please change the name  of the data members (For e.g. m_cache_depth, m_lru_function) following the same convention as other data members.

Reply:
changed the name of LRU to m_lru and cache_depth_specified to m_cache_depth_specified

Reply:
I meant this variable, as this is the data member of the class. Please change this to m_cache_depth 
```
int NGRAPH_TF_FUNCTION_CACHE_ITEM_DEPTH
```